openapi/openapi.yaml

```diff
@@ -31,26 +31,49 @@ paths:
       responses:
         "201":
           description: CREATED
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
         "400":
           description: BAD REQUEST
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
 
 
     get:
       summary: This gets all users
-      description: Return all users with paggination, could be sorted
+      description:  Return all users with paggination, could be sorted
       tags:
         - "users"
       operationId: getAllUsers
       parameters:
         - in: query
           name: sort
+          description: In order to execute *asc*, you need to specify in the search *id* or *username* parameter. In order to complete the *desk* sorting must be specified in the query parameter *-id*
           schema:
             type: string
-            enum: [ asc, desc ]
-
+            enum: [ "id", "-id", "username", "-username" ]
```

Comment:
We should rely on defined query parameters. In this case, you've hardcoded possible parameters for sorting without relying on it.

Please define query parameters.

users?age=50&name=Alex&id=5